README.md

```diff
@@ -1 +1,127 @@
 # cairo-coverage
+
+## Example
+
+Let's say you have a following program
+
+```rust
+pub enum Operation {
+    Add,
+    Multiply,
+}
+
+
+fn add(a: i32, b: i32) -> i32 {
+    a + b
+}
+
+fn multiply(a: i32, b: i32) -> i32 {
+    a * b
+}
+
+pub fn calculator(a: i32, b: i32, operation: Operation) -> i32 {
+    match operation {
+        Operation::Add => add(a, b),
+        Operation::Multiply => multiply(a, b),
+    }
+}
+
+```
+
+and you cover it with tests
+
+```rust
+#[test]
+fn calculator_add() {
+    assert_eq!(calculator(2, 3, Operation::Add), 5);
+    assert_eq!(calculator(-1, 1, Operation::Add), 0);
+}
+```
+
+When running with `cairo-coverage` you will get a coverage report in `.lcov` format:
+
+```lcov
+TN:
+SF:/path/to/your/project/src/lib.cairo
```

Comment:
Would be useful to have at least two-files example. Don't need to add it. Just maybe mention how second file would be added. 
I assume we just add another 
`SF:/path/to`

Reply:
Added 😄